RFS/src/main/java/org/opensearch/migrations/bulkload/lucene/version_7/DirectoryReader7.java

```diff
@@ -0,0 +1,34 @@
+package org.opensearch.migrations.bulkload.lucene.version_7;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.List;
+
+import org.opensearch.migrations.bulkload.lucene.LuceneDirectoryReader;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import shadow.lucene7.org.apache.lucene.index.DirectoryReader;
+
+@AllArgsConstructor
+public class DirectoryReader7 implements LuceneDirectoryReader {
```

Comment:
Effectively all the *7.java files will have direct references to `shadow.lucene7...` versions, much of this will be exactly the same for Lucene9 which I will convert after basic gut check that this is a good pattern.